setup.cfg

```diff
@@ -24,6 +24,7 @@ classifiers =
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
```

Comment:
pre-commit's `setup-cfg-fmt` forces this here. So should add Python 3.10 to testing.

Reply:
This is PR #41 now.